src/Common/AzurePSCmdlet.cs

```diff
@@ -434,6 +438,37 @@ protected bool IsVerbose()
             return verbose;
         }
 
+        protected void WriteSurvey()
+        {
+            HostInformationMessage msg = new HostInformationMessage()
+            {
+                Message = string.Format("Survey: How was your experience using Azure PowerShell?`\nRun ", ModuleName, ModuleVersion),
```

Comment:
```suggestion
                Message = "Survey: How was your experience using Azure PowerShell?`\nRun "
```
and use resource string please